packages/test-utils/src/wrapper.js

```diff
@@ -531,6 +531,116 @@ export default class Wrapper implements BaseWrapper {
     this.vnode = this.vm._vnode
   }
 
+  /**
+   * Sets element value and triggers input event
+   */
+  setValue (value: any) {
+    const el = this.element
+
+    if (!el) {
+      throwError('cannot call wrapper.setValue() on a wrapper without an element')
+    }
+
+    const tag = el.tagName
+    const type = this.attributes().type
+    const event = 'input'
+
+    if (tag === 'SELECT') {
+      throwError('wrapper.setValue() cannot be called on select')
```

Comment:
You can tell the user to call setSelected(). Also, can you change the errors to this format:

`wrapper.setValue() cannot be called on a <select> element. Use wrapper.setSelected() instead`